client/app/bundles/course/assessment/pages/AssessmentShow/AssessmentShowPage.tsx

```diff
@@ -51,7 +51,7 @@ const AssessmentShowPage = (props: AssessmentShowPageProps): JSX.Element => {
         .then(() => setSyncStatus('Synced'))
         .catch(() => setSyncStatus('Failed'));
```

Comment:
Shouldn't you use the constants `KODITSU_SYNC_STATUS.Synced` and `KODITSU_SYNC_STATUS.Failed` instead of `'Synced'` and `'Failed'` here?  
Same for line 46.

Reply:
done